README.md

```diff
@@ -65,16 +65,15 @@ your own Truffle language.
 
 Simple language is compatible to:
 
-* Truffle-Version: 0.31
-* GraalVM-Version: 0.31
+* Truffle-Version: 0.32
+* GraalVM-Version: 0.32
 
 ## Further information
 
-* [Truffle JavaDoc](http://lafo.ssw.uni-linz.ac.at/javadoc/truffle/latest/)
-* [Truffle on Github](http://github.com/graalvm/truffle)
-* [Graal on Github](http://github.com/graalvm/graal-core)
+* [Truffle JavaDoc](http://www.graalvm.org/truffle/javadoc/)
+* [Truffle on Github](https://github.com/oracle/graal/tree/master/truffle)
+* [Graal on Github](https://github.com/oracle/graal/tree/master/compiler)
 * [Truffle Tutorials and Presentations](https://wiki.openjdk.java.net/display/Graal/Publications+and+Presentations)
```

Comment:
link to here instead: https://github.com/oracle/graal/blob/master/docs/Publications.md